python-regression/tests/features/steps/transaction_steps.py

```diff
@@ -137,20 +122,24 @@ def make_transaction(node, arg_list):
 
 @step(r'"(\d+)" transactions are issued on "([^"]+)" with:')
 def issue_multiple_transactions(step, num_transactions, node):
-    transaction_hashes = []
-    args_copy = ()
+    # Placeholder values for seed if present
+    seed_value = ""
+    seed_type = ""
 
-    for iteration in range(int(num_transactions)):
+    for arg_index, arg in enumerate(step.hashes):
+        if arg['keys'] == "seed" and arg['type'] == "staticList":
+            seed_value = arg['values']
+            seed_type = arg['type']
 
-        if iteration == 0:
-            args_copy = api_utils.duplicate_arguments(step.hashes)
+    for iteration in range(int(num_transactions)):
+        seed = ""
+        if seed_value != "" and seed_type == "staticList":
+            seed = getattr(static, seed_value)[iteration]
```

Comment:
my python and aloe has grown weak...

What is this `static` object?

Reply:
This is the `static_values.py` file imported as `static`. It's fetching the seed from the list of seeds stored in `static_values.py`

Reply:
This is the `static_values.py` file imported as `static`